lms/templates/certificates/_accomplishment-rendering.html

```diff
@@ -51,7 +51,7 @@ <h3 class="accomplishment-signatories-title sr-only">${_("Noted by")}</h3>
 
                                     <h4 class="signatory-name hd-5">${signatory['name']}</h4>
                                     <p class="signatory-credentials copy copy-micro">
-                                        <span class="role">${signatory['title']|linebreaks}</span>
```

Comment:
If you're removing linebreaks here I think you can also remove the corresponding import statement above -- however we should confirm that this is the correct action because I believe we just recently added linebreaks to this template -- see SOL-1190